src/environments/route.ts

```diff
@@ -71,6 +71,15 @@ export const COLLECTIONS_ROUTES = {
   },
   GET_COLLECTION_CURATORS(name: string ) {
     return `${environment.apiURL}/users/curators/${encodeURIComponent(name)}`;
+  },
+  GET_COLLECTION_REPORT(collections: string[], date?: { start: string, end: string }) {
+    let route = `${environment.clarkReportsUrl}?output=csv&collection=${collections.join(',')}`;
```

Comment:
Route will need to be updated
```suggestion
    let route = `${environment.clarkReportsUrl}/reports?output=csv&collection=${collections.join(',')}`;
```